### ISSUE TITLE
Buildcraft Wrench, Crescent Hammer, etc don't work

### ISSUE THREAD

**Issue by da3dsoul** [planetguy32/RemainInMotion#45]
They just don't.

**Comment by da3dsoul** [comment]
I think I fixed it, I'm testing as soon as Minecraft stops crashing on startup from my attempts to not erase all previous carriages.

**Comment by da3dsoul** [comment]
Yay I fixed it.

**Comment by da3dsoul** [comment]
I lied only the Project:Red Screwdriver works...

**Comment by Korynkai** [comment]
Suggestion: Rather than look for other mods' tool implementations and supporting those, it would be preferred if you could create an interface that other mod developers can use to pull in functionality of your screwdriver. In this way, you prevent breakages in other mods (in my case: Powersuits) and ensure that the mods which would like to support your mod can support it without potentially causing ill-defined behaviour.

**Comment by da3dsoul** [comment]
I agree. We can make an interface, only the Project:Red screwdriver works as intended anyway as they all do strange things (like rotate).

**Comment by planetguy32** [comment]
If it's doing bad things we should remove the special wrench stuff, but IMO the last thing the world needs is another wrench API. I'd be inclined to stick to the Buildcraft and CoFH wrench APIs since they seem to be the most common.

**Comment by Korynkai** [comment]
Well, that would be one sane way of doing it, so long as RiM is inherently runtime dependent on whichever mod implements said tool API... 
On the other hand, if you're looking to eliminate runtime dependencies, you have to take into consideration if a mod inherits said tool interface specifically to support RiM and the mod which adds the aforementioned tool interface is not present (i.e. buildcraft), it would break functionality for the inheriting mod; therefore your own tool interface would be the way to go in this case.

**Comment by Blue64** [comment]
or, you could simply include a copy of the API you use (eg: Buildcraft Tool API) in your build and use it as a plan B if there's no newer version found.
